server-v2/api/studio/pkg/ecode/codes.go

```diff
@@ -46,6 +47,11 @@ func WithUnauthorized(err error, formatWithArgs ...interface{}) error {
 	return WithCode(ErrUnauthorized, err, formatWithArgs...)
 }
 
+func WithSessionMessage(err error, formatWithArgs ...interface{}) error {
+	ErrSessionWithMessage := newErrCode(CCUnauthorized, PlatformCode, 1, fmt.Sprintf("ErrSession::%s", err.Error()))
+	return WithCode(ErrSessionWithMessage, err, formatWithArgs...)
+}
```

Comment:
Only very general ones are put here, this is not recommended to be put here, this file should not be easily changed.

You just need to define the new ErrCode after line 12 `// Define you error code here`.